src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = u('.js-Slideshow-slides'),
+    slides = u('.js-Slideshow-slide'),
+    nextTrigger = u('.js-Slideshow-next'),
+    prevTrigger = u('.js-Slideshow-prev'),
+    currentCountElement = u('.js-Slideshow-current'),
+    totalCountElement = u('.js-Slideshow-total'),
```

Comment:
I believe to do what you were doing before in Umbrella land, you'd need to first run `element` through Umbrella (`u(element)`) and then use the `find` modifier (similar to jQuery). So altogether: `u(element).find('.js-Slideshow-slides')`